.idea/libraries/SBT__org_scala_lang_modules_scala_parser_combinators_2_12_1_0_4_jar.xml

```diff
@@ -1,11 +0,0 @@
-<component name="libraryTable">
```

Comment:
@tastulae you can ignore any of these files in the .idea folder during your review. There are about 10 of them at the top here. Check out the code below :)